src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -146,6 +146,22 @@ private static bool IsNamespaceName(NameSyntax name)
                 }
             }
 
+            if (name.IsUnmanaged && name.Parent.IsKind(SyntaxKind.TypeConstraint))
```

Comment:
Paging @CyrusNajmabadi on this little bit.

Reply:
This looks correct to me.  

Note: i don't like the .IsUnmanaged property as it sounds so ambiguous.  But it's parallel to the .IsVar property we have... so i guess we should keep it :)